src/libraries/System.Reflection.Emit/src/System/Reflection/Emit/ILGeneratorImpl.cs

```diff
@@ -817,4 +855,40 @@ internal LabelInfo(LabelHandle metaLabel)
         internal int _startDepth; // Stack labelStartDepth, with -1 meaning unknown.
         internal LabelHandle _metaLabel;
     }
+
+    internal sealed class Scope
+    {
+        public Scope(int offset, Scope? parent)
```

Comment:
```suggestion
        internal Scope(int offset, Scope? parent)
```

We should be consistent with the pattern here. Seems like if the type is `internal` all members should also be `internal`. I usually prefer to just mark the type `internal`, but it seems the more common pattern in the libraries is to mark members `internal` too.